src/components/RecentWorks.js

```diff
@@ -0,0 +1,269 @@
+import React, { useState } from 'react';
+
+import styled from 'styled-components';
+
+import Modal from 'react-modal';
+import CustomModal from './CustomModal';
+
+const Recent = styled.div`
+  width: 100%;
+
+  @media only screen and (max-width: 600px) {
+    width: 100%;
+    display: flex;
+    flex-direction: column;
+    
+  }
+`;
+
+const FirstRow = styled.div`
+  display: flex;
+  width: 90%;
+  margin: auto;
+
+  @media(max-width: 768px) {
+    display: grid;
+    grid-template-column: 1fr;
+  }
+`;
+
+const Divider = styled.div`
+  position: absolute;
+  width: 894px;
+  height: 0px;
+  left: 380px;
+  top: 952px;
+
+  border: 1px solid #212121;
+
+  /* Inside Auto Layout */
+
+  flex: none;
+  order: 1;
+  align-self: center;
+  flex-grow: 0;
+  margin: 24px 0px;
+
+  @media only screen and (max-width: 600px) {
+    width: 90%;
+    left: 10%;
+    top: 680px;
+    margin-left: -16px;
+  }
+
+  @media (min-width: 768px) and (max-width: 1024px) {
+    margin-left: -20px;
+  }
+`;
+
+const H1 = styled.h1`
+  position: static;
+  left: 0px;
+  top: 0px;
+
+  font-family: Crete Round;
+  font-style: normal;
+  font-weight: normal;
+  font-size: 40px;
+  line-height: 52px;
+  /* identical to box height, or 130% */
+
+  display: flex;
+  align-items: center;
+
+  /* Neutrals/N900 */
+
+  color: #091E42;
+
+  /* Inside Auto Layout */
+
+  flex: none;
+  order: 0;
+  align-self: center;
+  flex-grow: 0;
+  margin: 24px 0px;
+
+  @media only screen and (max-width: 600px) {
+    position: relative;
+    width: 100%;
+    left: -40px;
+    top: 0px;
+
+    font-family: Crete Round;
+    font-style: normal;
+    font-weight: normal;
+    font-size: 34px;
+    line-height: 52px;
+    /* or 130% */
+
+    display: flex;
+    align-items: center;
+
+    /* Theme 3 / Neutral 1 */
+
+    color: #1C1A19;
+
+    /* Inside Auto Layout */
+
+    flex: none;
+    order: 0;
+    flex-grow: 0;
+    margin: 0px 24px;
+  }
+`;
+
+const SeePro = styled.button`
+  background: #545454;
+`;
+const ProjectW = styled.div`
+  width: 90%;
+  margin: auto;
+  @media only screen and (max-width: 600px) {
+    width: 100%;
+  }
+`;
+
+const Proj = styled.div`
```

Comment:
- [ ] It would be better if we improve our semantics, let us use `<article>` instead of the `<div>`. Doing this helps make our code easier to maintain and readable

Reply:
This is a good request with a very good explanation. Good job!